components/script/dom/webglrenderingcontext.rs

```diff
@@ -2024,6 +2025,8 @@ impl CanvasContext for WebGLRenderingContext {
     }
 }
 
+impl WeakReferenceable for WebGLRenderingContext {}
```

Comment:
We also need to update Bindings.conf.

Reply:
it should fail, I suppose

Reply:
wait, according to CI https://github.com/servo/servo/actions/runs/15716818196/job/44288490593?pr=37512 If I set it in Binding.conf then I shouldn't impl the trait on the code. Right?

Reply:
I made a PR to fix the errore introduced in #37332 https://github.com/servo/servo/pull/37518